src/applications/vaos/containers/ReasonForAppointmentPage.jsx

```diff
@@ -92,30 +92,32 @@ export class ReasonForAppointmentPage extends React.Component {
             onBack={this.goBack}
             pageChangeInProgress={pageChangeInProgress}
           />
-          <AlertBox
-            headline="If you are experiencing a medical emergency:"
-            className="vads-u-margin-y--3"
-            content={
-              <ul>
-                <li>
-                  Call <a href="tel:911">911</a>,{' '}
-                  <span className="vads-u-font-weight--bold">or</span>
-                </li>
-                <li>
-                  Call the Veterans Crisis hotline at{' '}
-                  <a href="tel:8002738255">800-273-8255</a> and press 1,{' '}
-                  <span className="vads-u-font-weight--bold">or</span>
-                </li>
-                <li>
-                  Go to your nearest emergency room or VA medical center.{' '}
-                  <a href="/find-locations">
-                    Find your nearest VA medical center
-                  </a>
-                </li>
-              </ul>
-            }
-            status="warning"
-          />
+          <div aria-atomic="true" aria-live="assertive">
+            <AlertBox
+              status="warning"
+              headline="If you have an urgent medical need, please:"
+              className="vads-u-margin-y--3"
+              content={
+                <ul>
+                  <li>
+                    Call <a href="tel:911">911</a>,{' '}
+                    <span className="vads-u-font-weight--bold">or</span>
+                  </li>
+                  <li>
+                    Call the Veterans Crisis hotline at{' '}
+                    <a href="tel:8002738255">800-273-8255</a> and press 1,{' '}
+                    <span className="vads-u-font-weight--bold">or</span>
+                  </li>
+                  <li>
+                    Go to your nearest emergency room or VA medical center.{' '}
+                    <a href="/find-locations">
```

Comment:
This link should have `target="_blank"` and `rel="noopener noreferrer"` attributes, so that it opens in a new tab. I know you didn't change this, but as long as we're here we should fix it.

Reply:
Sure, I'll add those attributes to the link.

Reply:
Hi @jbaldoni, attributes added to the link. Ready for PR approval.